lib/rubocop/cop/rspec/example_wording.rb

```diff
@@ -29,14 +30,34 @@ module RSpec
       #   # good
       #   it 'does things' do
       #   end
+      #
+      # @example
+      #   # bad
+      #   it 'works' do
+      #   end
+      #
+      #   # good
+      #   it 'marks the task as done' do
+      #   end
+      #
+      # @example
+      #   # bad
+      #   it 'throws error' do
+      #   end
+      #
+      #   # good
+      #   it 'throws a not found error' do
+      #   end
       class ExampleWording < Base
         extend AutoCorrector
 
         MSG_SHOULD = 'Do not use should when describing your tests.'
         MSG_IT     = "Do not repeat 'it' when describing your tests."
+        MSG_INSUFFICIENT_DESCRIPTION = 'Your test should be more descriptive.'
 
         SHOULD_PREFIX = /\Ashould(?:n't)?\b/i.freeze
         IT_PREFIX     = /\Ait /i.freeze
+        INSUFFICIENT_EXAMPLES = ['works', 'throws error'].freeze
```

Comment:
It would be better if it could be changed with a configurable option, as suggested in https://github.com/rubocop/rubocop-rspec/issues/1337#issuecomment-1196650696

Reply:
DisallowedExamples:
    - works
    
My concern with adding it as a configurable option is the possibility of how big the list can get 😅 Are we anticipating it to get really big eventually?

Reply:
I think each project would have different examples of settings you want to disallowed, so wouldn't it be better to allow each project to configure its own settings?

Reply:
ah gotcha!!! makes sense!!